api/repo.go

```diff
@@ -536,6 +571,33 @@ func FindTagPath(exec boil.Executor, id int64) ([]*models.Tag, error) {
 	return ancestors, nil
 }
 
+func FindFileTreeWithOperations(exec boil.Executor, fileID int64) ([]*MFile, error) {
+
+	files := make([]*MFile, 0)
+
+	rsql := fmt.Sprintf(FILES_TREE_WITH_OPERATIONS, fileID, fileID, fileID)
+	rows, err := queries.Raw(exec, rsql).Query()
+	if err != nil {
+		return nil, NewInternalError(err)
+		defer rows.Close()
```

Comment:
This is unreachable code. you return just in the line above.
This line belongs below the if block as it's only meaningful when err is nil.